<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWidget</class>
 <widget class="QWidget" name="ClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1258</width>
    <height>1003</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font:&quot;Century Gothic&quot;;


</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_38">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_23">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Century Gothic&quot;;
color:rgb(255, 255, 255) ;</string>
         </property>
         <property name="text">
          <string>CLIENTS</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tabClient">
         <property name="minimumSize">
          <size>
           <width>1236</width>
           <height>517</height>
          </size>
         </property>
         <property name="palette">
          <palette>
           <active>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Text">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ButtonText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="PlaceholderText">
             <brush brushstyle="NoBrush">
              <color alpha="128">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Text">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ButtonText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="PlaceholderText">
             <brush brushstyle="NoBrush">
              <color alpha="128">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Text">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ButtonText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="PlaceholderText">
             <brush brushstyle="NoBrush">
              <color alpha="128">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Century Gothic&quot;;
color :rgb(255, 255, 255);

</string>
         </property>
         <property name="lineWidth">
          <number>3</number>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="autoScrollMargin">
          <number>50</number>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>150</number>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>150</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>50</number>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>50</number>
         </attribute>
         <column>
          <property name="text">
           <string>Nom client</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Prénom client</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Adresse</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Code Postal</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Ville</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Téléphone</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Siret</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Numéro client</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_21">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="ajoutClient">
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>15</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 9pt &quot;Gotham&quot; ;
font: bold;
color :rgb(255, 255, 0);
background-color: rgb(55, 55, 55);
border-style: solid;
border-color: rgb(50, 50, 50);
border-width: 2px;
border-radius: 9px;
</string>
           </property>
           <property name="text">
            <string>Ajouter</string>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="suppClient">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>15</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 9pt &quot;Gotham&quot; ;
font: bold;
color :rgb(255, 255, 0);
background-color: rgb(55, 55, 55);
border-style: solid;
border-color: rgb(50, 50, 50);
border-width: 2px;
border-radius: 9px;
</string>
           </property>
           <property name="text">
            <string>Supprimer</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true">font: 8pt &quot;Century Gothic&quot;;
color:rgb(255, 255, 255) ;</string>
         </property>
         <property name="title">
          <string>Informations</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_27">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_26">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_25">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_15">
                  <item>
                   <widget class="QLabel" name="label_11">
                    <property name="text">
                     <string>Nom</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="nomClient">
                    <property name="styleSheet">
                     <string notr="true">font: 8pt &quot;Century Gothic&quot;;
color: rgb(255, 255, 255);</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_20">
                  <item>
                   <widget class="QLabel" name="label_12">
                    <property name="text">
                     <string>Prenom</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="prenomClient">
                    <property name="styleSheet">
                     <string notr="true">font: 8pt &quot;Century Gothic&quot;;
color: rgb(255, 255, 255);</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_12">
                <item>
                 <widget class="QLabel" name="label_15">
                  <property name="text">
                   <string>Adresse</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="adresseClient">
                  <property name="styleSheet">
                   <string notr="true">font: 8pt &quot;Century Gothic&quot;;
color: rgb(255, 255, 255);</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_24">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_6">
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_14">
                    <item>
                     <widget class="QLabel" name="label_16">
                      <property name="text">
                       <string>Code postal</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="codePostalClient">
                      <property name="styleSheet">
                       <string notr="true">font: 8pt &quot;Century Gothic&quot;;
color: rgb(255, 255, 255);</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_13">
                    <item>
                     <widget class="QLabel" name="label_17">
                      <property name="text">
                       <string>Ville</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="villeClient">
                      <property name="styleSheet">
                       <string notr="true">font: 8pt &quot;Century Gothic&quot;;
color: rgb(255, 255, 255);</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_19">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_17">
                <item>
                 <widget class="QLabel" name="label_19">
                  <property name="text">
                   <string>Adresse mail</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="mailClient">
                  <property name="styleSheet">
                   <string notr="true">font: 8pt &quot;Century Gothic&quot;;
color: rgb(255, 255, 255);</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_16">
                <item>
                 <widget class="QLabel" name="label_18">
                  <property name="text">
                   <string>Numéro mobile</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="numClient">
                  <property name="styleSheet">
                   <string notr="true">font: 8pt &quot;Century Gothic&quot;;
color: rgb(255, 255, 255);</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_18">
                <item>
                 <widget class="QLabel" name="label_20">
                  <property name="text">
                   <string>Siret</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="siretClient">
                  <property name="styleSheet">
                   <string notr="true">font: 8pt &quot;Century Gothic&quot;;
color: rgb(255, 255, 255);</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Mot de passe</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="editpasswd">
                  <property name="styleSheet">
                   <string notr="true">font: 8pt &quot;Century Gothic&quot;;
color: rgb(255, 255, 255);</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
